### ISSUE TITLE
User dropdown not always working

### ISSUE THREAD

**Issue by danieldekerlegand** [mikevo/uno-heterogeneous-knowledge#67]
It seems that the user dropdown menu sometimes does not work. When the issue occurs while clicking on the dropdown, I can see the DOM being updated, but aria-expanded remains set to "false".
